src/gtk/dataview.cpp

```diff
@@ -4630,6 +4634,30 @@ gtk_dataview_motion_notify_callback( GtkWidget *WXUNUSED(widget),
     return FALSE;
 }
 
+static void evtDragDataGet(GtkWidget *widget, GdkDragContext *WXUNUSED(context),
```

Comment:
Minor, but it would be nice to use the usual convention for the GTK callbacks names, i.e. `wx_gtk_dataview_drag_data_get_callback` for this one.